tasks/nonograms/README.md

```diff
@@ -18,26 +18,26 @@ Here you can learn [how to solve the nonogram](https://nonograms-katana.fandom.c
 - initially, `body` in the index.html file must be empty (only script tag is allowed), **all necessary elements are generated using JS**
 - the design should be adaptive (or responsive) from _(500px <= width)_. It is acceptable to change the appearance for the mobile version (for example, hide the buttons in the burger menu)
 - the default size of the frame is 5x5. The clues are given at the top and left side of the grid. The sequence of numbers must be logically arranged and help the player solve the nonogram
-- every 5 cells in a rows and columns (both player's field and clues) should be divided by divider (bold line).
-- a player is able to fill in a square in the grid, using left mouse-click. On a click event, trigger the functionality of changing the color of the grid to dark (black). When player clicks on dark cell - it will change to empty (white).
-- end game when players fill all **black** squares correctly according to the hints. On a successful game solution, display "Great! You have solved the nonogram in ## seconds!"
+- for player's field: every 5 cells in a rows and columns should be divided by divider (bold line). For clues: every 5 cells in a **rows** should be divided by divider (for left clues); every 5 cells in a **columns** should be divided by divider (for top clues). Clues should be divided from player's field by bold line.
```

Comment:
I believe it's better to replace `for player's field` with `for game field`. What do you think?